src/views/footerView.jsx

```diff
@@ -197,7 +206,7 @@ export const ConferenceFooter = ({ isChatOpen, toggleChat }) => {
               {!isLocalVideoEnabled ? <CamOffIcon /> : <CamOnIcon />}
             </Button>
           ) : null,
-          isAllowedToPublish.video ? (
+          isAllowedToPublish.video && pluginRef.current && pluginRef.current.isSupported() ? (
```

Comment:
pluginRef.current?.isSupported()

Reply:
since we are already checking for pluginRef.current before isSupported, I think its not required to add the optional part

Reply:
Yes I mean we can remove two conditions and have only one.